tests/4.5/target_update/test_target_update_if.c

```diff
@@ -71,7 +70,7 @@ for(count=0; count< 4;count++){
   change_flag = init_b();
   #pragma omp target update if (change_flag) to(b[:N]) //update b=4 for all odd iterations
 
-  #pragma omp target //map(from: a,b,c) map(tofrom: isHost)//mapping from to counter default tofrom mapping on b
+  #pragma omp target //map(from: a,b,c) //mapping from to counter default tofrom mapping on b
```

Comment:
remove the map(from) and potentially the comment. It's ok if you have a comment here explaining what's the purpose, but we need to fix this wording I believe.